system/LINK/bin/fix_auth

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# description: ManageIQ fix_auth launcher
+#
+
+if [ -d /var/www/miq/vmdb ] ; then
+  cd /var/www/miq/vmdb
+else
+  # remove symbolic links to script
+  SCRIPT=$(readlink "$0")
+  # dir name
+  SCRIPTPATH=$(dirname "${SCRIPT:-$0}")
+  # rails root
+  cd "${SCRIPTPATH}/../../../vmdb"
```

Comment:
@Fryguy I did this in response to feedback from @jrafanie - he wanted to make sure this use case worked when the bin was a symbolic link.

I just want to be able to run this from the appliance or from a dev setup. I think this is overkill but hey - what ever floats people's boat